app/src/main/java/org/mozilla/focus/utils/ThreadUtils.java

```diff
@@ -1,45 +0,0 @@
-/* -*- Mode: Java; c-basic-offset: 4; tab-width: 20; indent-tabs-mode: nil; -*-
- * This Source Code Form is subject to the terms of the Mozilla Public
- * License, v. 2.0. If a copy of the MPL was not distributed with this
- * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
-
-package org.mozilla.focus.utils;
-
-import android.os.Handler;
-import android.os.Looper;
-
-import java.util.concurrent.ExecutorService;
-import java.util.concurrent.Executors;
-
-import edu.umd.cs.findbugs.annotations.SuppressFBWarnings;
-
-public class ThreadUtils {
```

Comment:
Do we just use kotlin coroutines instead, and no longer care about posting to UI/background threads? If we're going to use threads in the future is this worth keeping?

Reply:
My plan is to just use kotlin coroutines instead.

The value of this file is that it 1) creates a centralized background thread for the app and 2) makes the calling background methods simpler in Java.

For 1), coroutines creates a shared ThreadPool we can post to, we can use existing primitives that spawn threads (like AsyncTask), or we can create an app-wide Executor that we can post to. For 2) we're using kotlin and lambdas make this much simpler.